marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/jobInfo/jobInfo.sjs

```diff
@@ -0,0 +1,41 @@
+/*
+  Copyright 2012-2019 MarkLogic Corporation
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+'use strict';
+
+var model;
+
+let res = {
+  "modelName": model,
+  "jobId": null,
+  "jobTime": null
+};
+
+let query = [cts.collectionQuery(model), cts.fieldRangeQuery("datahubCreatedOn", "<=", fn.currentDateTime(), "score-function=reciprocal")];
+let seq = fn.subsequence(cts.search(cts.andQuery(query)), 1, 1);
+
+if (fn.count(seq) > 0) {
+  let docUri = xdmp.nodeUri(seq);
+
+  res.jobTime = xdmp.documentGetMetadataValue(docUri, "datahubCreatedOn");
+
+  let jobIds = xdmp.documentGetMetadataValue(docUri, "datahubCreatedByJob");
+  if (jobIds) {
+    jobIds = jobIds.split(" ");
+    res.jobId = jobIds[jobIds.length - 1];
+  }
+}
+
+res;
```

Comment:
Since we're adding this to the DHF project, I think we need to have tests for it as well in the project. I'll spend a little time with this on Monday to determine what the right mix of ML-based unit tests and Java-based integration tests would be appropriate (I'm currently leaning towards just unit tests).